public/instances.json

```diff
@@ -0,0 +1,72 @@
+{
+  "instances": [
+    {
+      "name": "Contrib",
```

Comment:
> Why not simply use the `name` as the key and list them all at root level? This would spare you from having to navigate the data through `instancesData.instances`, for example 🙂

Reply:
Done with https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/e19159902f322d612c392ae1eea25f4f0c4dfa08